features/step_definitions/work_download_steps.rb

```diff
@@ -1,3 +1,15 @@
+When /^I download "(.*?)"$/ do |title|
+  work = Work.find_by(title: title)
+  visit work_url(work)
+  step %{I follow "HTML"}
+end
+
+Then /^I should see the inspiring parent work link$/ do
+  parent = Work.find_by(title: "Worldbuilding")
+  inspired_link = "<a href=\"#{work_url(parent)}\">#{parent.title.html_safe}</a>"
```

Comment:
It looks like the issue was for a problem that occurs specifically with external works, not works. Could we test that case as well (or instead)?

Reply:
Fair enough, test added!